docs/index.md

```diff
@@ -2,19 +2,31 @@
 
 A Simple and Comprehensive Vulnerability Scanner for Containers and other Artifacts, Suitable for CI.
 
-<img src="imgs/usage.gif" width="700">
-<img src="imgs/usage1.png" width="600">
-<img src="imgs/usage2.png" width="600">
-
-## Abstract
-
+# Abstract
 `Trivy` (`tri` pronounced like **tri**gger, `vy` pronounced like en**vy**) is a simple and comprehensive vulnerability scanner for containers and other artifacts.
 A software vulnerability is a glitch, flaw, or weakness present in the software or in an Operating System.
 `Trivy` detects vulnerabilities of OS packages (Alpine, RHEL, CentOS, etc.) and application dependencies (Bundler, Composer, npm, yarn, etc.).
 `Trivy` is easy to use. Just install the binary and you're ready to scan. All you need to do for scanning is to specify a target such as an image name of the container.
 
+<img src="imgs/overview.png" width="700">
+
+Trivy can be run in two different modes:
+
+- [Standalone](https://aquasecurity.github.io/trivy/latest/modes/standalone/)
+- [Client/Server](https://aquasecurity.github.io/trivy/latest/modes/client-server/)
```

Comment:
We should use relative paths to markdown files / sections in order to let MkDocs to transform them into valid links. Absolute paths may also break preview run locally.

```suggestion
- [Standalone](./modes/standalone.md)
- [Client/Server](./modes/client-server.md)
```